### PR TITLE
Fix aliyun tf output rendering and refine documents

### PR DESCRIPTION
**Pull request by aylei**
Signed-off-by: Aylei <rayingecho@gmail.com>

### What problem does this PR solve? 
Fix the output render error when kubernetes is not successfully created in aliyun.

### What is changed and how it works?
TF scripts changed.

### Check List 

Tests 

 - Manual test (Passed)

@AstroProfundis @tennix @IzabelWang  PTAL

### CODE REVIEW

File: deploy/aliyun/ack/outputs.tf [hunk 1]
```diff
@@ -1,11 +1,11 @@
 output "cluster_id" {
   description = "The id of the ACK cluster."
-  value       = "${alicloud_cs_managed_kubernetes.k8s.id}"
+  value       = "${alicloud_cs_managed_kubernetes.k8s.*.id}"
 }
 
 output "cluster_name" {
   description = "The name of ACK cluster"
-  value       = "${alicloud_cs_managed_kubernetes.k8s.name}"
+  value       = "${alicloud_cs_managed_kubernetes.k8s.*.id}"
```
Comment (tennix) [inline deploy/aliyun/ack/outputs.tf]: Why change cluster name to id?

Comment (aylei) [reply deploy/aliyun/ack/outputs.tf]: Good catch, this is a typo😅

**Comment by aylei** [comment]
@tennix Thanks for your review, I've fix a merge conflict just now, could you please take a look again?

**Pull request merged**